pkg/daemon/rpm-ostree.go

```diff
@@ -260,10 +261,11 @@ func (r *RpmOstreeClient) Rebase(imgURL, osImageContentDir string) (changed bool
 func runGetOut(command string, args ...string) ([]byte, error) {
 	glog.Infof("Running captured: %s %s", command, strings.Join(args, " "))
 	cmd := exec.Command(command, args...)
-	cmd.Stderr = os.Stderr
+	var stderr bytes.Buffer
+	cmd.Stderr = &stderr
 	rawOut, err := cmd.Output()
 	if err != nil {
-		return nil, errors.Wrapf(err, "error running %s %s: %s", command, strings.Join(args, " "), string(rawOut))
+		return nil, errors.Wrapf(err, "error running %s %s: %s%s", command, strings.Join(args, " "), string(rawOut), stderr.string())
```

Comment:
```suggestion
		return nil, errors.Wrapf(err, "error running %s %s: %s%s", command, strings.Join(args, " "), string(rawOut), stderr.String())
```

CI errors are from a missing capital letter :slightly_smiling_face:. The `String` function on `bytes.Buffer` should [start with a capital letter](https://golang.org/pkg/bytes/#Buffer.String).

Reply:
sigh, i fixed it locally and didn't do git add before ammend